manager/vulnerabilities_handler.py

```diff
@@ -20,13 +20,13 @@ class CvesListView(ListView):
     def __init__(self, list_args, query_args, uri, args, ids_only=False):
         join_type = JOIN.INNER
         cve_count = CveAccountData.systems_affected
-        if 'show_all' in args and args['show_all']:
+        if ('show_all' in args and args['show_all']) or (args['affecting'] is not None and args['affecting'][1]):
```

Comment:
affecting may not be in args?

Reply:
`affecting` is always in the args due to argument parsing on line 216.